workspace/apps/pidp/src/app/features/organization-info/pages/endorsements/endorsements.page.ts

```diff
@@ -211,6 +213,37 @@ export class EndorsementsPage
       .subscribe();
   }
 
+  public getStatus(endorsementRequestStatus: EndorsementRequestStatus): string {
+    return (
+      endorsementRequestsLabelText[endorsementRequestStatus] ?? 'Requested'
+    );
+  }
+
+  public isEndorsementRequested(
+    endorsementRequestStatus: EndorsementRequestStatus,
+  ): boolean {
+    return ![
```

Comment:
Can we define this array in the constants. In later phases if the same defined array is used in other components/places then if any changes comes to array we can modify the array in the constants and the components business logic remains same.

Reply:
Sounds good